nextcloud-inotifyscan

```diff
@@ -150,7 +150,7 @@ def watch_instances(instances):
       # path_prefix is the virtual folder recognized by Nextcloud
       # data_prefix is the actual filesystem folder
       # both should point to the same logical folder
-      path_prefix = '/'+user+'/files'
+      path_prefix = '/'+user+'/files/'
```

Comment:
This works but is not necessary. I reverted this change to make all prefixes consistently end without trailing `/`.

Reply:
Thanks! You know the logic better than I do - glad it was helpful:)